src/pages/app/searching_project_user/ProjectInfoPage.js

```diff
@@ -219,6 +219,7 @@ const ProjectInfoPage = (props) => {
           ))}
         </ul>
       </div>
+      //Somthing is wrong in this section, detail:"benefits". That makes the page broken
```

Comment:
@kenpham4real 
plz resolve that asap, so i can code :"(((, becuz my task is inherited by this page :"((

Reply:
I'm working on it now.